### PR TITLE
404 repair

### PR DESCRIPTION
**Pull request by SeekingYang**
127-130

### CODE REVIEW

File: versioned_docs/version-0.15/best-practices/star-schema-benchmark.md [hunk 1]
```diff
@@ -162,4 +162,4 @@ The following test report is based on Doris [branch-0.15](https://github.com/apa
     >
     > Note 4: Parallelism means query concurrency, which is set by `set parallel_fragment_exec_instance_num=8`.
     >
-    > Note 5: Runtime Filter Mode is the type of Runtime Filter, set by `set runtime_filter_type="BLOOM_FILTER"`. ([Runtime Filter](http://doris.incubator.apache.org/master/en/administrator-guide/runtime-filter.html) function has a significant effect on the SSB test set. Because in this test set, The data from the right table of Join can filter the left table very well. You can try to turn off this function through `set runtime_filter_mode=off` to see the change in query latency.)
+    > Note 5: Runtime Filter Mode is the type of Runtime Filter, set by `set runtime_filter_type="BLOOM_FILTER"`. ([Runtime Filter](https://doris.apache.org/docs/0.15/administrator-guide/runtime-filter/) function has a significant effect on the SSB test set. Because in this test set, The data from the right table of Join can filter the left table very well. You can try to turn off this function through `set runtime_filter_mode=off` to see the change in query latency.)
```
Comment (JNSimba) [inline versioned_docs/version-0.15/best-practices/star-schema-benchmark.md:165]: url uses relative path

File: versioned_docs/version-0.15/extending-doris/logstash.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ under the License.
 
 This plugin is used to output data to Doris for logstash, use the HTTP protocol to interact with the Doris FE Http interface, and import data through Doris's stream load.
 
-[Learn more about Doris Stream Load ](http://doris.apache.org/master/zh-CN/administrator-guide/load-data/stream-load-manual.html)
+[Learn more about Doris Stream Load ](https://doris.apache.org/docs/0.15/administrator-guide/load-data/stream-load-manual/)
```
Comment (JNSimba) [inline versioned_docs/version-0.15/extending-doris/logstash.md:31]: same as above

File: versioned_docs/version-0.15/installing/install-deploy.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Doris, as an open source MPP architecture OLAP database, can run on most mainstr
 
 > Note 2: Number of FE nodes
 > 
-> 1. FE roles are divided into Follower and Observer. (Leader is an elected role in the Follower group, hereinafter referred to as Follower, for the specific meaning, see [Metadata Design Document](./internal/metadata-design).)
+> 1. FE roles are divided into Follower and Observer. (Leader is an elected role in the Follower group, hereinafter referred to as Follower, for the specific meaning, see [Metadata Design Document](https://doris.apache.org/community/design/metadata-design).)
```
Comment (JNSimba) [inline versioned_docs/version-0.15/installing/install-deploy.md:77]: same as above

Comment (SeekingYang) [reply versioned_docs/version-0.15/installing/install-deploy.md:77]: this url cannot use relative path

**Pull request merged**